integrations/drift/drift.go

```diff
@@ -90,7 +90,6 @@ func (api driftAPIProduction) request(method string, endpoint string, payload []
 	req.Header.Set("Content-Type", "application/json")
 	client := &http.Client{}
 	resp, err := client.Do(req)
-	defer resp.Body.Close()
```

Comment:
Why move this? If you move it that way, it will never be closed in case of error where it was closed before.